### PR TITLE
feat: logout return_to at mfa

### PR DESCRIPTION
**Pull request by Benehiko**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: src/routes/login.ts [hunk 1]
```diff
@@ -57,12 +57,21 @@ export const createLoginRoute: RouteCreator =
...
+        (
+          await frontend
+            .createBrowserLogoutFlow({
+              cookie: req.header("cookie"),
+              returnTo:
+                (return_to && return_to.toString()) ||
+                loginFlow.return_to ||
+                "",
+            } as { cookie: string; returnTo: string })
+            .catch(() => ({ data: { logout_url: "" } }))
```
Comment (jonas-jonas) [inline src/routes/login.ts:71]: Why not try { .. } catch () { }?

File: src/routes/login.ts [hunk 2]
```diff
@@ -57,12 +57,21 @@ export const createLoginRoute: RouteCreator =
...
+      return (
+        (
+          await frontend
+            .createBrowserLogoutFlow({
+              cookie: req.header("cookie"),
+              returnTo:
+                (return_to && return_to.toString()) ||
+                loginFlow.return_to ||
+                "",
+            } as { cookie: string; returnTo: string })
```
Comment (jonas-jonas) [inline src/routes/login.ts:70]: Why is the cast neccessary?

Comment (Benehiko) [reply src/routes/login.ts:70]: ah, forgot to remove it

File: src/routes/login.ts (hunk 1 above)
Comment (Benehiko) [reply src/routes/login.ts:71]: can do, but does it change anything? :p

Comment (jonas-jonas) [reply src/routes/login.ts:71]: You don't mix .then/.catch with await :) it's a codestyle nit...

**Review by jonas-jonas — APPROVED** [review]

**Pull request merged**